### PR TITLE
Crawler

### PR DESCRIPTION
**Pull request by esl-lewis**
Uploaded script for automating executable downloads, small reorg of directories.

### CODE REVIEW

**Comment by sebastien-rettie** [comment]
Thanks Emily! Do we need to add all the empty files in the `dataset/malware/` directory?

File: dataset/malware/chrome_crawler.py [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+
+# Just a test to see if we can grab names+urls of first 10 apps, yes we can 
+"""
+for row in table.findAll('div', class_='c-globalCard lg:u-col-3 md:u-col-3 sm:u-col-2 c-productCard u-flexbox-column c-productCard-detailed'):
+    app_name = row.find('div', class_='c-productCard_info')
+    print(app_name.h3.string)
+    app_page_link = row.find('a', class_='c-productCard_link')
+    print(app_page_link['href'])
+"""
```
Comment (sebastien-rettie) [inline dataset/malware/chrome_crawler.py]: Would it be possible to cleanup the commented out blocks of code to avoid confusion?

**Comment by esl-lewis** [comment]
> 
> 
> Thanks Emily! Do we need to add all the empty files in the `dataset/malware/` directory?

We do not! Thanks for catching that

**Review by sebastien-rettie — APPROVED** [review]

File: dataset/malware/chrome_crawler.py (hunk 1 above)
Comment (esl-lewis) [reply dataset/malware/chrome_crawler.py]: Agreed- this was a previous attempt using chrome so isn't in the current commit.

**Pull request merged**